staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -344,30 +374,100 @@ func TestShouldDelegateList(t *testing.T) {
 			}
 		}
 	}
-	consistentListFromCacheOverrides := map[opts]bool{}
-	for _, recursive := range []bool{true, false} {
-		consistentListFromCacheOverrides[opts{Recursive: recursive}] = false
-		consistentListFromCacheOverrides[opts{Limit: 100, Recursive: recursive}] = false
-	}
+
+	// Exacts and continue on RV observed by cache.
+	listFromCacheSnapshotOverrides := map[opts]bool{}
+	listFromCacheSnapshotOverrides[opts{Recursive: true, Continue: continueOnOldRV}] = false
+	listFromCacheSnapshotOverrides[opts{Recursive: true, Limit: 100, Continue: continueOnOldRV}] = false
+	listFromCacheSnapshotOverrides[opts{Recursive: true, ResourceVersion: "0", Continue: continueOnOldRV}] = false
+	listFromCacheSnapshotOverrides[opts{Recursive: true, ResourceVersion: "0", Limit: 100, Continue: continueOnOldRV}] = false
+	listFromCacheSnapshotOverrides[opts{Recursive: true, ResourceVersion: oldRV, Limit: 100}] = false
+	listFromCacheSnapshotOverrides[opts{Recursive: true, ResourceVersion: oldRV, ResourceVersionMatch: metav1.ResourceVersionMatchExact}] = false
+	listFromCacheSnapshotOverrides[opts{Recursive: true, ResourceVersion: oldRV, ResourceVersionMatch: metav1.ResourceVersionMatchExact, Limit: 100}] = false
+
+	exactOnCacheRVOverrides := map[opts]bool{}
```

Comment:
Separation of `exactOnCacheRVOverrides` and `exactOnCacheRVOverrides` was not intentional. I noticed it just extending the test to cover continuation on cacheRV.

Reply:
nvm, must have been a artifact of bad precedense.